templates/workloads/services/lb-web/cf.yml

```diff
@@ -45,7 +45,7 @@ Conditions:
     !Equals [!Ref HTTPSEnabled, true]
   HasAddons: # If a bucket URL is specified, that means the template exists.
     !Not [!Equals [!Ref AddonsTemplateURL, ""]]
-  HTTPRootPath: # If we're using path based routing and use the root path, we have some special logic
+  IsDefaultRootPath: # If we're using path based routing and use the root path, we have some special logic
```

Comment:
nit: This comment is not super helpful as we don't describe the special logic, can we remove it?